index.html

```diff
@@ -19,120 +20,182 @@
+                    <div class="card_content ">
+                        <div>
+                            <h3 class="card_title ">Project name goes here</h3>
+                        </div>
+                        <ul class="span_button">
+                            <li><span href="" class="btn card_btn1 ">HTML/CSS</span></li>
+                            </li> <span href="" class="btn card_btn2 ">Ruby on Rails</span></li>
+                            </li><span href="" class="btn card_btn3 ">JavaScript</span></li>
+                        </ul>
+                        <button class="card_btn"><span>See this project</span>
+                        <img src="images/icon.svg" height="12px" ></button>
                     </div>
-                    <ul class="span_button">
-                        <li><span href="" class="btn card_btn1 ">HTML/CSS</span></li>
-                        </li> <span href="" class="btn card_btn2 ">Ruby on Rails</span></li>
-                        </li><span href="" class="btn card_btn3 ">JavaScript</span></li>
-                    </ul>
-                    <button class="card_btn"><span>See this project</span>
+                </div>
+                <div class="grid-item">
+                    <img class="card_image " src="images/1.png ">
+                    <div class="card_content ">
+                        <div>
+                            <h3 class="card_title ">Project name goes here</h3>
+                        </div>
+                        <ul class="span_button">
+                            <li><span href="" class="btn card_btn1 ">HTML/CSS</span></li>
+                            </li> <span href="" class="btn card_btn2 ">Ruby on Rails</span></li>
+                            </li><span href="" class="btn card_btn3 ">JavaScript</span></li>
+                        </ul>
+                        <button class="card_btn"><span>See this project</span>
                         <img src="images/icon.svg" height="12px" ></button>
+                    </div>
                 </div>
             </div>
     </section>
-    <div class="about">
-        <h2>About</h2>
-        <p>Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip ea eiusmod</p>
-        <a href="" class="button"><span>See this project</span>
-        <img src="images/icon1.svg" height="16px" ></a>
-    </div>
+    
     <section class="skills_section">
+        <div class="about">
+            <h2>About</h2>
+            <p>Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip ea eiusmod</p>
+            <a href="" class="button"><span>See this project</span>
+            <img src="images/icon1.svg" height="16px" ></a>
+        </div>
```

Comment:
Kindly modify the space between the `Get Started` heading and the background image above it in order to match the design template.

Your design:
![image](https://user-images.githubusercontent.com/20554583/206877785-8712de5e-f15c-4f2e-9131-dcadfda29f5e.png)


Figma design:
![image](https://user-images.githubusercontent.com/20554583/206877778-7aa4115f-5c2a-4415-9d53-c0e516465557.png)